main.js

```diff
@@ -1,77 +1,79 @@
 'use strict';
-const { Telegraf } = require('telegraf');
+const Telegraf = require('telegraf');
```

Comment:
```suggestion
const telegraf = require('telegraf');
```
Именование `camelCase`